static/app/components/profiling/profileHeader.tsx

```diff
@@ -13,13 +13,27 @@ import space from 'sentry/styles/space';
 import {Event} from 'sentry/types';
 import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
 import {getTransactionDetailsUrl} from 'sentry/utils/performance/urls';
+import {isSchema, isSentrySampledProfile} from 'sentry/utils/profiling/guards/profile';
 import {
   generateProfileDetailsRouteWithQuery,
   generateProfileFlamechartRouteWithQuery,
 } from 'sentry/utils/profiling/routes';
 import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
-import {useProfileGroup} from 'sentry/views/profiling/profileGroupProvider';
+import {useProfiles} from 'sentry/views/profiling/profilesProvider';
+
+import {ProfileInput} from '../../utils/profiling/profile/importProfile';
```

Comment:
Any reason to use a relative import here? We don't really use it in many places.

Reply:
Nope, just auto import messing this one up, I wish it had some option to always do absolute imports tbh

Reply:
Fixed in https://github.com/getsentry/sentry/pull/44097